tuxemon/formula.py

```diff
@@ -333,3 +333,9 @@ def rematch(
     # restore hp evolved monsters
     for mon in opponent.monsters:
         mon.current_hp = mon.hp
+
+
+def synch(player: NPC, value: int, total: int) -> float:
```

Comment:
I'd prefer "sync".

https://www.merriam-webster.com/dictionary/sync

Reply:
Thanks for the dictionary, I'll update